lib/layouts/settings/chat_list_panel.dart

```diff
@@ -34,7 +34,8 @@ class ChatListPanel extends StatelessWidget {
       headerColor = Theme.of(context).backgroundColor;
       tileColor = Theme.of(context).colorScheme.secondary;
     }
-    if (SettingsManager().settings.skin.value == Skins.iOS && isEqual(Theme.of(context), oledDarkTheme)) {
+    if (SettingsManager().settings.skin.value == Skins.iOS && Theme.of(context).backgroundColor == Colors.black
```

Comment:
Hey can u make all these booleans into a helper? It's a lot of the same reused code and we should make a helper function for it